var/spack/repos/builtin/packages/madgraph5amc/package.py

```diff
@@ -50,6 +52,8 @@ class Madgraph5amc(Package):
     patch('madgraph5amc-2.7.3.atlas.patch', level=0, when='@2.7.3.py3+atlas')
     patch('madgraph5amc-2.7.3.patch', level=0, when='@2.8.0~atlas')
     patch('madgraph5amc-2.8.0.atlas.patch', level=0, when='@2.8.0+atlas')
+    patch('madgraph5amc-2.7.3.patch', level=0, when='@2.8.1~atlas')
+    patch('madgraph5amc-2.8.0.atlas.patch', level=0, when='@2.8.1+atlas')
```

Comment:
I think we can simplify this patch stuff by removing `@X.Y.Z` from the when args since the patch is always needed for all versions.